world/concert_with_human.sdf

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" ?>
+<sdf version="1.4">
+
+    <world name="default">
+
+        <!-- simulation time support  -->
+        <plugin name="xbot2_clock" filename="libxbot2_gz_clock.so"/>
+
+        <gravity>0 0 -9.81</gravity>
+        
+        <include>
+            <file>/home/user/tum_integration_ws/catkin_ws/src/human-gazebo/models/human_work</file>
```

Comment:
open todo

Reply:
Fixed in https://github.com/manuelvogel12/human-gazebo/commit/784d761426e586b9f72af9e7cfb75aba7cf350c1, no issue needed.